smithy-swift-codegen-test/smithy-build.json

```diff
@@ -10,7 +10,8 @@
                     "gitRepo": "https://github.com/smithy-lang/smithy-swift.git",
                     "author": "Amazon Web Services",
                     "homepage": "https://docs.amplify.aws/",
-                    "swiftVersion": "5.7.0"
+                    "swiftVersion": "5.7.0",
+                    "mergeModels": false
```

Comment:
This new option in `smithy-build.json` allows you to control merging of models.  I added it here even though it defaults to `false`.